examples/server/tests/features/server.feature

```diff
@@ -0,0 +1,39 @@
+Feature: llama.cpp server
+
+  Background: The server is started and ready to accept prompts
+    When wait for the server to be started
+    Then wait for the server to be healthy
+
+  Scenario: Health endpoint
+    Given an health liveness probe
+    Then the server must be healthy
+
+  Scenario Outline: run a completion request
+    Given a prompt <prompt>
+    When we request a completion
+    Then tokens are predicted
+
+    Examples: Prompts
+      | prompt       |
+      | I believe    |
+      | Write a joke |
+
+  Scenario Outline: run a completion on the OAI endpoint
+    Given a system prompt <system_prompt>
+    And a user prompt <user_prompt>
+    And a model <model>
+    When we request the oai completions endpoint
+    Then the oai response contains completion tokens
+
+    Examples: Prompts
+      | model       | system_prompt               | user_prompt                         |
+      | tinyllama-2 | You are ChatGPT.            | Say hello                           |
+      | tinyllama-2 | You are a coding assistant. | Write the fibonacci function in c++ |
+
+
+  Scenario: Health endpoint during processing with concurrent requests
```

Comment:
@ngxson Here is a first example of concurrent requests, happy to have your feedback to simplify it.

Reply:
Thanks. Overall it's good here, just some small suggestions regarding the naming:
- `wait for all slots processing` maybe confused (i.e. slots start or stop processing?), I'd suggest to change it to `wait N slots to become busy` where N can be 1, 2,....
- Also, for steps that check service health, can we change them to `the server health is...` ? This way we reflect the `health` keyword in step's name

Reply:
Thanks @ngxson, I have tried to simplify and factorize the steps, is the new naming is clearer ?

Reply:
yes it's more clear now, thanks!